.github/workflows/e2e.yml

```diff
@@ -52,198 +52,7 @@ jobs:
         uses: ./.github/actions/e2e_validate_generated_app
         with:
           PROJECT_KEY: "retail-app-demo"
-  run-generator-retail-app-no-ext:
```

Comment:
In the future, will we want to add some E2E tests similar to these ones?

Reply:
Yup, we probably want to add E2E tests in the future with different extensions.